breadbox/breadbox/compute/celery.py

```diff
@@ -1,13 +1,29 @@
-from celery import Celery
+from celery import Celery, Task
 import os
 from logging import getLogger
 
+from breadbox.logging import GCPExceptionReporter
 
-log = getLogger(__name__)
 
 rhost = os.getenv("REDIS_HOST", "localhost")
+breadbox_env = os.getenv("BREADBOX_ENV", "dev")
+
+log = getLogger(__name__)
+exception_reporter = GCPExceptionReporter(breadbox_env=breadbox_env)
+
+class LogErrorsTask(Task):
+    def on_failure(self, exc, task_id, args, kwargs, einfo):
+        """Override the default behavior on task failure to also report to our GCP error logs.""" 
+
+        # It would be nice if we could report some of the information we do have here
+        # (like task_id, args, etc.) to error_reporter. However, error_reporter is really 
+        # only set up to log context in the format of a HTTP request. 
+        exception_reporter.report(request=None, status_code=None, user=None)
+        super(LogErrorsTask, self).on_failure(exc, task_id, args, kwargs, einfo)
```

Comment:
just as a fyi, the above looks like how calling a superclass's method worked in python 2. It should work but in python 3+ you can just write:
```
        super().on_failure(exc, task_id, args, kwargs, einfo)
  ```
https://docs.python.org/3.7/library/functions.html#super